internal/pkg/template/templates/workloads/partials/cf/vpc-connector.yml

```diff
@@ -0,0 +1,39 @@
+ServiceSecurityGroup:
+  Type: AWS::EC2::SecurityGroup
+  Metadata:
+    'aws:copilot:description': 'A security group for your addons to allow ingress'
+  Properties:
+    GroupDescription: !Join [ '', [ !Ref AppName, '-', !Ref EnvironmentName, !Ref WorkloadName ] ]
```

Comment:
Should we write the description such as 'A security group for your RDS Aurora addons to allow ingress from VPC private subnets'

Reply:
I didn't want to say something specific to RDS because they could be using it also for elasticache clusters, opensearch clusters.

Maybe this?
```
'A security group for your addon resources to allow ingress from your service'
```

Reply:
updated, let me know what you think!